101-cognitive-services-Computer-vision-API/metadata.json

```diff
@@ -2,9 +2,9 @@
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "itemDisplayName": "Cognitive Services Computer Vision API",
   "description": "A template for creating a new Cognitive Services Computer Vision API",
-  "summary": "A template for creating a new Cognitive Services Computer Vision API. Use this template as the framework for your custom deployment.",
-  "githubUsername": "olandese",
-  "dateUpdated": "2019-11-25",
+  "summary": "A template for creating a new Cognitive Services Computer Vision API. Use this template as the framework for your custom deployment. This template was originally created by Marco Mansi (@olandese).",
+  "githubUsername": "HamzaBoukraa",
```

Comment:
is your intent to take ownership of this sample?

Reply:
@bmoore-msft not at all, it was not my intention. I just was Fixing errors and warnings of https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=27562&view=results

I thought the metadata is for this commited version : date + commiter.

Reply:
It clearly says that it is the original author , so I reverted the change related to githubUserName !
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#githubusername